env.d/production

```diff
@@ -0,0 +1,24 @@
+# Python
+PYTHONUNBUFFERED=1
+
+# Django
+DJANGO_SETTINGS_MODULE=funmooc.settings
+DJANGO_CONFIGURATION=ContinuousIntegration
```

Comment:
You can't run with the Production settings because they store media files in S3, expect SSL, a CDN, etc.

Reply:
True. Maybe there is something more relevant than `ContinuousIntegration` settings...

Reply:
ContinuousIntegration settings are not used by anything else so I can just rename them? It seems a good name though since that's what is used in the CI?

But what's a better name? Or there is also a Test settings class that is not use either...

Reply:
I cannot find something better than this...